<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>374</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>人工智能课程代码编辑器-选择题模式</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/badge2.png</normaloff>:/img/badge2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="importBtn">
        <property name="text">
         <string>导入答案</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportCodeBtn">
        <property name="text">
         <string>导出代码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportAnswerBtn">
        <property name="text">
         <string>导出答案</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back2HomeBtn">
        <property name="text">
         <string>返回主菜单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">#groupContent {
	background-image: url(:/img/badge4.png);
	background-repeat: no-repeat;
	background-attachment: fixed;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="groupContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>508</width>
           <height>298</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="importAction">
   <property name="text">
    <string>导入</string>
   </property>
   <property name="statusTip">
    <string>导入python文件</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="statusTip">
    <string>另存为python文件</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存为python文件</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ui.rsc.qrc"/>
 </resources>
 <connections/>
</ui>
